package/src/oneSignal.ts

```diff
@@ -10,20 +10,48 @@ const DEFAULT_MODULE_SCRIPT_ID = 'react-onesignal-module';
  */
 const ONE_SIGNAL_SCRIPT_SRC = 'https://cdn.onesignal.com/sdks/OneSignalSDK.js';
 
+/**
+ * Maps The Options Object Into A String For The Module Script
+ * @param o the options object
+ * @param indent 
+ */
+const mapOptionsObject = (o: any, indent: number = 0) => {
```

Comment:
Correct me if I am wrong, but wouldn't JSON.stringify(o, null, 1) also do the work?
If you don't want opening and closing braces you can do JSON.stringify(o, null, 1).slice(1).slice(0,-1).

Reply:
Very good @gax97!

This would do the job and it's MUCH simpler :)